hosts/galileo/__init__.py

```diff
@@ -0,0 +1,395 @@
+################################################################################
+# hosts/galileo/__init__.py: Galileo class
+#
+# Utility functions for managing SPICE kernels while working with Galileo data
+# sets.
+################################################################################
+import numpy as np
+import unittest
+import os.path
+
+import julian
+import textkernel
+import spicedb
+import cspyce
+import oops
+
+from oops.body import Body
+
+TOUR = (2003 - 2000) * 365 * 86400      # Rough ET dividing Saturn from Jupiter
+
+################################################################################
+# Routines for managing the loading of C and SP kernels
```

Comment:
SP -> SPK?

Reply:
K stands for kernel, so CK / SPK kernel is redundant